web3/utils/datastructures.py

```diff
@@ -95,6 +99,15 @@ def __eq__(self, other):
         else:
             return False
 
+    @staticmethod
+    def serializeHexBytes(value):
+        return value.hex() if isinstance(value, HexBytes) else value
+
+    @property
+    def serialized(self):
+        serializers = compose(AttributeDict.serializeHexBytes)
```

Comment:
Compose with just one function is a noop.

Reply:
Ah right, haha i thought this would be helpful for adding future serializers but I realize now its just garbage hahaha